sdk/identity/tests.yml

```diff
@@ -1,36 +1,12 @@
 trigger: none
 
-resources:
```

Comment:
Can we delete the dockerfile from `eng/containers` as well?

Reply:
I think we may need it in the future, so do you mind if we leave it around for now?